Controllers/Backend/ViewSnapshots.php

```diff
@@ -55,4 +58,68 @@ public function listAction()
             ['success' => true, 'data' => $data, 'total' => $total]
         );
     }
+
+    /**
+     * @throws ParameterMissingException
+     * @throws Exception
+     */
+    public function diffAction()
+    {
+        $sessionFrom = $this->Request()->getParam('sessionFrom');
+        $stepFrom = $this->Request()->getParam('stepFrom');
+        $sessionTo = $this->Request()->getParam('sessionTo', $sessionFrom);
+        $stepTo = $this->Request()->getParam('stepTo');
+
+        if (empty($sessionFrom)) {
+            throw new ParameterMissingException('sessionFrom is missing');
+        }
+
+        if (empty($stepFrom)) {
+            throw new ParameterMissingException('stepFrom is missing');
+        }
+
+        if (empty($sessionTo)) {
```

Comment:
This check is not necessary, because it will never be reached `if (empty($sessionFrom)) {` applies.

Reply:
True. I'll fix that.